import_certs.sh

```diff
@@ -4,5 +4,5 @@ cd `dirname $0`
 
 for certfile in $(ls *.crt *.cer 2> /dev/null)
 do
-    keytool -importcert -file $certfile -alias $certfile -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -noprompt
```

Comment:
@nleach999 why Keystore flag was removed ?

Reply:
-cacerts does the same thing without having to resolve the path to the cacerts keystore.

Reply:
oh, good to know, didn't knew about that option, thanks !